openml/extensions/sklearn/extension.py

```diff
@@ -65,7 +65,10 @@
 
 
 class SklearnExtension(Extension):
-    """Connect scikit-learn to OpenML-Python."""
+    """Connect scikit-learn to OpenML-Python.
+       The estimators which use this extension must be scikit-learn compatible',
```

Comment:
```suggestion
       The estimators which use this extension must be scikit-learn compatible,
```

Reply:
The doc build is failing due to a broken link error. I checked this with Pieter:


- The link is broken because it's not in the main branch yet. The documentation is here: https://github.com/openml/openml-python/tree/develop/docker
- As soon as the next merge to main happens, the links should work